### PR TITLE
Added example file, updated .gitignore and README.

### PR DESCRIPTION
**Pull request by dhsrocha**
Commit for attending #36

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/nodejs-http-client?pullRequest=41) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/nodejs-http-client?pullRequest=41) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/nodejs-http-client?pullRequest=41) it.</sub>

File: README.md [hunk 1]
```diff
@@ -96,6 +96,8 @@ echo "sendgrid.env" >> .gitignore
 source ./sendgrid.env
 ```
 
+You cand use the ```.env_example``` file located at "examples" folder. Just copy it to a ```.env``` folder ou can put on your project's root.
```
Comment (mbernier) [inline README.md]: Can you please change: 
> You cand use the ```.env_example``` file located at "examples" folder. Just copy it to a ```.env``` folder ou can put on your project's root.

To this: 

> You can copy the `.env_example` file located in the "examples" folder to `.env` in your project's root.
> ```cp examples/.env_sample .env```
> Then add your API Key into the file

**Comment by mbernier** [comment]
@dhsrocha can you please sign the CLA? We can't merge your code without it.

**Comment by mbernier** [comment]
I just noticed you have signed in another repo - I know it's a bit redundant., and I am sorry for the trouble, our legal team has asked that people sign at each repo.

**Comment by thinkingserious** [comment]
Hello @dhsrocha,<br /><br/>Thanks again for the PR!<br />                                              <br />We want to show our appreciation by sending                                              you some swag. Could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Comment by mbernier** [comment]
@dhsrocha 
If you are interested, we would love the opportunity to talk to you about Hacktoberfest and our API libraries.

Our agenda would be: Explore what you liked and is there anything we can do to improve? 

You can grab a time [on my calendar](https://calendly.com/matt-bernier/hacktoberfest-follow-up) that works for you and we can have a chat on Google Hangout or Skype. If you prefer, you can email me using my GitHub username at my company’s domain.

Thank you so much,

Matt Bernier - @mbernier - SendGrid Developer Experience Product Manager
Elmer Thomas - @thinkingserious - SendGrid Developer Experience Engineer
@SendGrid

**Pull request merged**